app/controlplane/api/workflowcontract/v1/crafting_schema.proto

```diff
@@ -144,3 +147,49 @@ message PolicyAttachment {
     string value = 2 [(buf.validate.field).required = true];;
   }
 }
+
+// Represents a policy to be applied to a material or attestation
+message Policy {
```

Comment:
To avoid import cycles. In particular `policy.proto` needed material types, and crafting_schema needed policies. We could extract the common part to a third proto, but that would require several unrelated changes throughout all the code.